src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt

```diff
@@ -76,22 +76,22 @@ class SarifSerializerTest {
     @Test
     fun `serialize kotlin data class to minified json`() {
         val actual = SarifSerializer.toMinifiedJson(sarifSchema210)
-        val expected = File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
+        val expected = getResource("/test.sarif.json").readText()
         assertEquals(expected.replace("\\s".toRegex(), ""), actual)
     }
 
     @Test
     fun `serialize kotlin data class to json`() {
-        val actual = SarifSerializer.toJson(sarifSchema210).lines()
-        val expected = File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readLines()
+        val actual = SarifSerializer.toJson(sarifSchema210)
+        val expected = getResource("/test.sarif.json").readText()
         assertEquals(expected, actual)
     }
 
     @Test
     fun `deserialize json into kotlin data class`() {
-        val actual = SarifSerializer.fromJson(
-            File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
-        )
+        val actual = SarifSerializer.fromJson(getResource("/test.sarif.json").readText())
         assertEquals(sarifSchema210, actual)
     }
-}
\ No newline at end of file
+}
+
+private fun getResource(path: String): Reader = SarifSerializerTest::class.java.getResourceAsStream(path)!!.reader()
```

Comment:
One very long line, wrap?
```suggestion
private fun getResource(path: String): Reader =
     SarifSerializerTest::class.java.getResourceAsStream(path)!!.reader()
```
(Please double check before commit, I wrote it on my phone and code editor was funny)

Reply:
I did the change using Idea to avoid issues :P